paper.tex

```diff
@@ -556,7 +556,7 @@ \subsubsection{Messages Publishment}
 \section{Conclusions}
 By combining the established standards and openly-developed specifications, this paper proposed an autonomous publish/subscribe model design to serve as a vendor and industry-neutral platform, automating the trading of digital assets and services. It was built with blockchain network, immutable audit trails, and contracts with an integrated decentralized identity system, to ensure the authenticity of all participants, enable secure communication and flexible trading mechanisms.
 
-In the subscription-based data trading, there are some issues that worth to discuss and improve. Firstly, reduce the Gas usage of trading models, as we mentioned in Section~\ref{smart_contract_evaluation}, the key exchange process consumes a lot Gas due to the frequent store operation. Therefore, it is worth studying how zero-knowledge protocol could used to exchange key with lower cost, and even validate the key within transfers. Second, brokers are irreplaceable either from the view of architecture design or trading procedures, they are the key to the stable system. However, the issues such as the minimum number of brokers needed to operation the system, and the evaluation of adopting gossip protocol among brokers to gossipped trading status, which enables other brokers to handle trading process when on crashed.
+In the subscription-based data trading, there are some issues that worth to discuss and improve. Firstly, reduce the Gas usage of trading models, as we mentioned in Section~\ref{smart_contract_evaluation}, the key exchange process consumes a lot Gas due to the frequent store operation. Therefore, it is worth discussing that how zero-knowledge protocol could used to exchange key with lower cost, and even validate the key within transfers. Second, brokers are irreplaceable either from the view of architecture design or trading procedures, they are the key to the stable system. However, the issues such as the minimum number of brokers needed to operation the system, and the evaluation of adopting gossip protocol among brokers to gossipped trading status, which enables other brokers to handle trading process when on crashed.
```

Comment:
Second, brokers are irreplaceable either from the view of architecture design or trading procedures, so they are the key to the stable system.